introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-chatbot-openchatkit.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    image_uri = "763104351884.dkr.ecr.us-west-2.amazonaws.com/djl-inference:0.21.0-deepspeed0.8.3-cu117"
</pre><p>plz use image retriever to get the image as it could be multiple regions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3943/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>